src/main/java/org/junitpioneer/jupiter/RepeatFailedTestExtension.java

```diff
@@ -72,8 +73,8 @@ private static FailedTestRepeater repeaterFor(ExtensionContext context) {
 
 		private FailedTestRepeater(int maxRepetitions) {
 			this.maxRepetitions = maxRepetitions;
-			this.repetitionsSoFar = 0;
-			this.exceptionsSoFar = 0;
+			repetitionsSoFar = 0;
+			exceptionsSoFar = 0;
```

Comment:
The `this.` are unnecessary, but IMO they improve readability by keeping the three assignments aligned for easy top-down reading.

Reply:
I can follow your argumentation and suggest that @aepfli disables the rule

Reply:
hmm, is there really a rule, i cant find a violation in sonarqube

Reply:
Again save actions